RELEASES.md

```diff
@@ -30,12 +30,12 @@
 - Fix: ensure temp files are on the same volume (#257)
 - Intercept global package installations (#248)
 - Fix: make npx compatible with prelrease versions of npm (#239)
-- Fix: make `notion deactivate` work infallibly, without loading any files (#237)
+- Fix: make `volta deactivate` work infallibly, without loading any files (#237)
 - Fix: make `"npm"` key optional in `package.json` (#233)
 - Fix: publish latest Notion version via self-hosted endpoint (#230)
 - Fix: eliminate excessive fetching and scanning for exact versions (#227)
-- Rename `notion use` to `notion pin` (#226)
-- Base filesystem isolation on `NOTION_HOME` env var (#224)
+- Rename `volta use` to `volta pin` (#226)
+- Base filesystem isolation on `VOLTA_HOME` env var (#224)
```

Comment:
Should we be changing the historical entries in `RELEASES.md`?  On the one hand, it may reduce confusion by not presenting the "wrong" name, but on the other, it may *introduce* confusion by changing history (for example, Notion 0.2.0 will **not** change the `VOLTA_HOME` variable).

Reply:
Good catch; I was intending not to! Pushed a commit that fixes it.